### PR TITLE
Improve view debuggroup, reduce columns

### PR DESCRIPTION
**Pull request by chmst**
Pull Request for Issue #38828 .

### Summary of Changes
This is an alternative PR to #39335, it improves the debug view of  permissions for usergroups.
Same code as in https://github.com/joomla/joomla-cms/pull/37807 plus https://github.com/joomla/joomla-cms/pull/39481

### Testing Instructions
Look at usergroups, permissions for a group

### Actual result BEFORE applying this Pull Request
grafik

### Expected result AFTER applying this Pull Request
grafik

### Link to documentations
Please select:
- [ ] Documentation link for docs.joomla.org: <link>
- [x] No documentation changes for docs.joomla.org needed

- [ ] Pull Request link for manual.joomla.org: <link>
- [x] No documentation changes for manual.joomla.org needed

### CODE REVIEW

**Comment by brianteeman** [comment]
+100 for consistency with the other tables

**Comment by richard67** [comment]
@chmst Please fix the code style errors reported here: https://ci.joomla.org/joomla/joomla-cms/60418/1/8 .

**Comment by richard67** [comment]
Something I don’t understand: The description says 

> Same code as in https://github.com/joomla/joomla-cms/pull/37807 plus https://github.com/joomla/joomla-cms/pull/39481

Both mentioned PR‘s have been merged into 4.2-dev, so if that statement was true, this PR here should show zero changes.

**Comment by chmst** [comment]
@richard67 the logic of the newly added code is the same, the layout and indetion of lines itself is slightly different.

**Comment by Quy** [comment]
Please fix conflicts.

File: administrator/components/com_users/tmpl/debuggroup/default.php [hunk 1]
```diff
@@ -26,12 +26,52 @@
...
+                        $text   = Text::_('COM_USERS_DEBUG_EXPLICIT_DENY');
+                    elseif ($check === null) :
+                        $class  = 'text-danger icon-minus-circle';
+                        $button = 'btn-warning';
+                        $text   = Text::_('COM_USERS_DEBUG_IMPLICIT_DENY');
+                    endif; ?>
+                    <div class="d-inline p-2">
+                        <?php echo Text::_($action[1]); ?>
+                        <span class="<?php echo $class; ?>" aria-hidden="true"></span>
+                        <span class="visually-hidden"><?php echo Text::_($text); ?></span>
```
Comment (Quy) [inline administrator/components/com_users/tmpl/debuggroup/default.php]:
```suggestion
                        <span class="visually-hidden"><?php echo $text; ?></span>
```

Comment (chmst) [reply administrator/components/com_users/tmpl/debuggroup/default.php]: thx

File: administrator/components/com_users/tmpl/debuggroup/default.php [hunk 2]
```diff
@@ -61,9 +101,9 @@
                                 <?php echo $this->escape(Text::_($item->title)); ?>
                             </th>
                             <td>
-                                <?php echo LayoutHelper::render('joomla.html.treeprefix', ['level' => $item->level + 1]) . $this->escape($item->name); ?>
+                                <?php echo LayoutHelper::render('joomla.html.treeprefix', array('level' => $item->level + 1)) . $this->escape($item->name); ?>
```
Comment (Quy) [inline administrator/components/com_users/tmpl/debuggroup/default.php]:
```suggestion
                                <?php echo LayoutHelper::render('joomla.html.treeprefix', ['level' => $item->level + 1]) . $this->escape($item->name); ?>
```

**Comment by HLeithner** [comment]
This pull request has been automatically rebased to 5.0-dev. No new features will be merged into Joomla! 4.3 series. Joomla! 4.4 series is a bridge release to make migration from Joomla! 4 to 5 as smooth as possible.

**Comment by ChrisHoefliger** [comment]
I have tested this item :white_check_mark: successfully on 8b8058a2a154208886cea0a97ff4d84dfed94067<br /><br />Tested successfully on localhost Joomla 4.3.4 PBF 2023<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/39525">issues.joomla.org/tracker/joomla-cms/39525</a>.</sub>

**Comment by rachelwalraven** [comment]
I have tested this item :white_check_mark: successfully on 8b8058a2a154208886cea0a97ff4d84dfed94067<br /><br />Tested succesfully on joomla 5.0 php 8.1.22<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/39525">issues.joomla.org/tracker/joomla-cms/39525</a>.</sub>

**Comment by RickR2H** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/39525">issues.joomla.org/tracker/joomla-cms/39525</a>.</sub>

**Comment by RickR2H** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/39525">issues.joomla.org/tracker/joomla-cms/39525</a>.</sub>

**Comment by RickR2H** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/39525">issues.joomla.org/tracker/joomla-cms/39525</a>.</sub>

**Comment by bembelimen** [comment]
Revert<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/39525">issues.joomla.org/tracker/joomla-cms/39525</a>.</sub>

**Comment by bembelimen** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/39525">issues.joomla.org/tracker/joomla-cms/39525</a>.</sub>

**Comment by HLeithner** [comment]
this has many RTCs^^ thanks

**Pull request merged**